docs/lang/articles/advanced/dataclass.md

```diff
@@ -22,19 +22,16 @@ class Sphere:
     center: vec3
     radius: ti.f32
 ```
-This creates the *exact* same type as using `ti.types.struct()`:
+This is the same equivalent as using `ti.types.struct()`:
 
 ```python
 Sphere = ti.types.struct(center=vec3, radius=ti.f32)
 ```
-The `@ti.dataclass` decorator converts the annotated members in the Python class to members in the resulting struct type. In both of the above examples, you end up with the same struct field.
+The `@ti.dataclass` decorator converts the annotated members in the `Python class` to members in the resulting `Struct Type`. In both of the above examples, you end up with the same struct field.
```

Comment:
@neozhaoliang @vai9er Can you check the use of the inline code format here? I thought inline code should be used for excerpts from code snippets.